content/geometry/object-vehicle/vehicle-structure.adoc

```diff
@@ -1386,30 +1385,34 @@ image::Grp_Mirror_Side_Joint_Left.svg[,600]
 |===
 
 
-== Grp_Mirror_Side_Joint_Right_<side_mirror_joint_right_idx> (T)
+== Grp_Mirror_Side_Mounting_Right_<side_mirror_mounting_right_idx> (T)
```

Comment:
For this sequence, I'd really follow the structure as in the structure overview:
1. Side mirror mounting left -> Side mirror joint left -> Side mirror view left
2. Side mirror mounting right -> Side mirror joint right -> Side mirror view right

Reply:
Updated chapter sequence and removed "optional" from structure overview